lib/modules/dosomething/dosomething_rogue/dosomething_rogue.cron.inc

```diff
@@ -56,9 +56,9 @@ function dosomething_rogue_retry_failed_reportbacks() {
       } else {
         $data = [
           [
-            'rogue_reportback_item_id' => $task->rogue_item_id,
+            'rogue_event_id' => $task->rogue_event_id,
             'status' => $task->status,
-            'reviewer' => $task->reviewer,
+            'event_type' => 'post_photo',
```

Comment:
@katiecrane for now, yes! Kept this as `post_photo` since I remember when we were talking about the signup business that we shouldn't add `update_signup` as the `event_type` yet and not get too wrapped up in the naming quiet yet. Happy to change though if we think this is a good time to discuss `event_type`s!